packages/shared/lib/core/nfts/actions/updateNftInAllAccountNfts.ts

```diff
@@ -8,6 +9,13 @@ export function updateNftInAllAccountNfts(accountIndex: number, nftId: string, p
         }
         const nft = state[accountIndex].find((_nft) => _nft.id === nftId)
         if (nft) {
+            const downloadUrl = nft.downloadUrl
+            void getIpfsUri({ hash: downloadUrl }).then((ipfsUri) => {
```

Comment:
Should we check if the `downloadUrl` is a ipfs link before calling `getIpfsUri` for every nft? wdyt?